spec/models/dhs1426_pdf_spec.rb

```diff
@@ -152,40 +163,54 @@
         primary_member_self_employed: "Yes",
         primary_member_first_employed_employer_name: "AA Accounting",
         primary_member_first_employed_pay_interval_hourly: "Yes",
-        # primary_member_first_employed_hours_interval_weekly: nil,
```

Comment:
seems like these un-commenting changes are perhaps not related to this specific PR?